build/tools/Automation/Python/Constants.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace Oryx.Microsoft.Automation.Python
+{
+    public class Constants
```

Comment:
Are we able to rename this class to `PythonConstants.cs` so that we don't have to use a namespace prefix every time we want to reference a constant in this project? (similar comment for .NET)

Reply:
Updated